webpack.config.js

```diff
@@ -29,7 +30,13 @@ module.exports = e => {
     resolve: {
         // Add '.ts', '.tsx', and '.mjs' as resolvable extensions.
         // '.mjs' is used by the graphql library which is a dependency of aws-amplify
-        extensions: [".mjs", ".ts", ".tsx", ".js", ".json"]
+        extensions: [".mjs", ".ts", ".tsx", ".js", ".json"],
+        alias: {
```

Comment:
Did the aliases end up working? It looks like your classes are still referencing `../../api`, etc.